src/core/energy_storage/electric_storage.jl

```diff
@@ -168,7 +169,7 @@ end
     soc_min_fraction::Float64 = 0.2
     soc_min_applies_during_outages::Bool = false
     soc_init_fraction::Float64 = off_grid_flag ? 1.0 : 0.5
-    can_grid_charge::Bool = off_grid_flag ? false : true
+    can_grid_charge::Bool = off_grid_flag || dc_coupled ? false : true
```

Comment:
Have you tested that allowing grid to charge bess does work when dc coupled?

Reply:
Do you mean overriding this default and setting can_grid_charge=true for a dc coupled scenario? I haven't made a test for this specifically that e.g. checks the result of grid to storage var. I was originally going to not allow grid charging when dc coupled but then switched to this, so should probably create that test now.